tests/unit/test_publisher_backlist_service.py

```diff
@@ -0,0 +1,139 @@
+            "Digitization status": ["recjlYWETJ1aOiWTG"],
+            "DRB_File Location": "https://drive.google.com/open?id=1Cy1LF_G8OyRHp0UpsV2zDv4gNiXIIh7T",
+            "DRB_Ready to ingest": True,
+            "Access types": ["recVXZOA70wwC1ciY"],
+            "Next step (from Project steps)": ["Ready to ingest"],
+            "Rights status (from Rights status linked record)": [
+                "public domain",
+                "cleared by rights team",
+            ],
+            "Publisher (from Project)": ["University of Michigan Press"],
+            "Project status": ["In process"],
+            "Status (from Digitization status)": ["Publisher digitized"],
+            "Last Modified": "2024-06-20T17:15:35.000Z",
+            "DRB_Record ID": "recPTBrKmZnsguK8M",
+            "Project Name (from Project)": ["Test Backlist"],
+            "Access type in DRB (from Access types)": ["Full access"],
+        },
+    },
+    {
+        "id": "recRvPlHdLkM0KIem",
+        "createdTime": "2023-12-21T19:06:11.000Z",
+        "fields": {
+            "Author(s)": "Broadbooks, Harold E.",
+            "Title": "Life history and ecology of the chipmunk, Eutamias amoenus, in eastern Washington",
+            "Pub Date": "1958",
+            "Copyright holder": "Publisher",
+            "Rights status linked record": ["recX5aollpn3ubbkM", "recLXN9DU6RJehpDB"],
+            "Project steps": ["recUI0frKtIAtVZND"],
+            "File ID 1": "39015032124508",
+            "Digitial copy at press?": "yes",
+            "Digital copy received": True,
+            "Project": ["recDhurlgY5mQ4F8p"],
+            "OCLC": "1892309",
+            "Rights classification by publisher (UMP only)": "A",
+            "Rights review / PD or renewal": True,
+            "Hathi rights code": "und",
+            "DRB Rights Classification": "public domain",
+            "Digitization status": ["recjlYWETJ1aOiWTG"],
+            "DRB_File Location": "https://drive.google.com/open?id=19ibZ2CtAzIflL4N1tf3NoBonr2-rxfQA",
+            "DRB_Ready to ingest": True,
+            "Access types": ["recVarWgjQaYXJZDM"],
+            "Next step (from Project steps)": ["Ready to ingest"],
+            "Rights status (from Rights status linked record)": [
+                "public domain",
+                "cleared by rights team",
+            ],
+            "Publisher (from Project)": ["University of Michigan Press"],
+            "Project status": ["In process"],
+            "Status (from Digitization status)": ["Publisher digitized"],
+            "Last Modified": "2024-06-20T17:15:35.000Z",
+            "DRB_Record ID": "recRvPlHdLkM0KIem",
+            "Project Name (from Project)": ["Test Backlist"],
+            "Access type in DRB (from Access types)": [
+                "Limited access/login for read & download"
+            ],
+        },
+    }]
+                
+    def test_get_records(self, test_airtable_records, testService, mocker):
```

Comment:
I recommend formatting this test for readability. What I have found works well is the following format:
```
test setup, mocks, environment variables 
new line
function/method that is under test
new line
assertions
```